Makefile

```diff
@@ -23,7 +23,7 @@ test:
 .PHONY: test
 
 site:
-	git clone https://github.com/github/cli.github.com.git "$@"
+	(which gh 2>/dev/null && gh repo clone github/cli.github.com "$@") || git clone https://github.com/github/cli.github.com.git "$@"
```

Comment:
For simplicity sake, let's just declare that `site:` is dependent on `bin/gh` and use `bin/gh repo clone github/cli.github.com` unconditionally here.

This will not be exercised in CI because there we set up `site` manually via a step in the `releases.yml` workflow.

Reply:
ah fair; I missed that the `site` target was not in use in CI.

Reply:
turns out the `site` target is used

Reply:
I made a wrong call here, sorry: I thought that the `site:` target will be a no-op because the `site/` directory will have already existed. However, the `bin/gh` prerequisite brought the timestamps into the mix and then it didn't matter anymore that `site/` exists because it was seen as "out of date"